Content.Server/Chat/Systems/ChatSystem.cs

```diff
@@ -295,6 +300,56 @@ private void OnGameChange(GameRunLevelChangedEvent ev)
 
     #region Private API
 
+    public void SendCollectiveMindChat(EntityUid source, string message, bool hideChat)
+    {
+        if (!TryComp<CollectiveMindComponent>(source, out var sourseCollectiveMindComp))
+            return;
+
+        var clients = Filter.Empty();
+        var mindQuery = EntityQueryEnumerator<CollectiveMindComponent, ActorComponent>();
+        while (mindQuery.MoveNext(out var uid, out var collectMindComp, out var actorComp))
+        {
+            if (collectMindComp.Channel == sourseCollectiveMindComp.Channel)
+            {
+                clients.AddPlayer(actorComp.PlayerSession);
+            }
+        }
+
+        var admins = _adminManager.ActiveAdmins
+            .Select(p => p.ConnectedClient);
+        string messageWrap;
+        string adminMessageWrap;
+
+        messageWrap = Loc.GetString("chat-manager-send-collective-mind-chat-wrap-message",
+            ("message", message),
+            ("channel", sourseCollectiveMindComp.Channel));
+
+        adminMessageWrap = Loc.GetString("chat-manager-send-collective-mind-chat-wrap-message-admin",
+            ("source", source),
+            ("message", message),
+            ("channel", sourseCollectiveMindComp.Channel));
+
+        _adminLogger.Add(LogType.Chat, LogImpact.Low, $"CollectiveMind chat from {ToPrettyString(source):Player}: {message}");
+
+        _chatManager.ChatMessageToManyFiltered(clients,
+            ChatChannel.CollectiveMind,
+            message,
+            messageWrap,
+            source,
+            hideChat,
+            true,
+            sourseCollectiveMindComp.ChannelColor);
+
+        _chatManager.ChatMessageToMany(ChatChannel.CollectiveMind,
```

Comment:
just add `admins` to `clients` and use a single method (you can change the messageWrap with a ternary operator)